api/src/main/resources/public/api/v1/schema/jobspec.schema.json

```diff
@@ -26,6 +26,9 @@
         "type": "string"
       }
     },
+    "schedules": {
+      "type": "array"
```

Comment:
I thought we could add
```
"items": { "$ref": "schedulespec.schema.json" }
```

However, our validation does not support these kind of references.

Reply:
Darn; does a newer version support it?

If not, can we at least copy-pasta?

Reply:
> Darn; does a newer version support it?

I don't quite understand the Metronome implementation. I don't think the JSON schema expects schemas to be loaded from custom paths but via URIs.

> If not, can we at least copy-pasta?
I'd say so.